charts/sn-platform-slim/values.yaml

```diff
@@ -305,6 +305,7 @@ auth:
       autoInitSecret: false
     tls:
       enabled: false
+    statusCheck: false
```

Comment:
Why do we set its default value `false` ?

Reply:
@tuteng pls update the default value to `true`